stdlib/Cargo.toml

```diff
@@ -15,8 +15,25 @@ rust-version = "1.67"
 bench = false
 doctest = false
 
+[[test]]
+name = "stdlib"
+path = "tests/main.rs"
+
 [dependencies]
 assembly = { package = "miden-assembly", default-features = false, path = "../assembly", version = "0.6" }
 
+[dev-dependencies]
+blake3 = "1.3.3"
+num-bigint = "0.4.3"
+processor = { package = "miden-processor", path = "../processor", version = "0.6", features = ["internals"], default-features = false }
+proptest = "1.1.0"
+rand-utils = { package = "winter-rand-utils", version = "0.6" }
+sha2 = "0.10.6"
+sha3 = "0.10.6"
+test-case = "3.0.0"
```

Comment:
We shouldn't need to import such things as `proptest`, `rand-utils`, and `test-case` as they can be imported from `test-utils`.

Reply:
Looks like this merge https://github.com/0xPolygonMiden/miden-vm/pull/788/commits/08a32bd9bcc5fb28d3ee2f27473e5d1dc46a45f7 brought them in again. Just fixed in 24af1f8f